mlir/include/mlir/Dialect/EmitC/IR/EmitC.td

```diff
@@ -409,9 +409,9 @@ def EmitC_ExpressionOp : EmitC_Op<"expression",
     int32_t v7 = foo(v1 + v2) * (v3 + v4);
     ```
 
-    The operations allowed within expression body are emitc.add, emitc.apply,
-    emitc.call, emitc.cast, emitc.cmp, emitc.div, emitc.mul, emitc.rem and
-    emitc.sub.
+    The operations allowed within expression body are `emitc.add`,
+    `emitc.apply`, `emitc.call`, `emitc.cast`, `emitc.cmp`, `emitc.div`,
```

Comment:
```suggestion
    `emitc.apply`, `emitc.call_opaque`, `emitc.cast`, `emitc.cmp`, `emitc.div`,
```

Reply:
I think I'd need add one more? Since both `call` and `call_opaque` should work?

Reply:
Yes, there are also missing some other operations which were added recently. Make sure to this in sync with https://github.com/llvm/llvm-project/blob/7a0acccd81df268dc7ad4c0358c42552789f19b4/mlir/include/mlir/Dialect/EmitC/IR/EmitC.td#L430 if you want to add more.

Reply:
Not without further changes, see

https://github.com/llvm/llvm-project/blob/7a0acccd81df268dc7ad4c0358c42552789f19b4/mlir/include/mlir/Dialect/EmitC/IR/EmitC.td#L430-L434

which is used in `ExpressionOp::verify()`. But if we change this, this should be done in a follow up.

Reply:
Ah, hm, it doesn't have `call` in there, so I guess it should be send in a separate PR along with e.g. logical operations.

Reply:
> But if we change this, this should be done in a follow up.

I can add those later today.

Reply:
>I can add those later today.

yeah, go ahead if you want, I just started learning about mlir/emitc, so it'll take me awhile. Though given that I use emitc directly I'd likely need them.

I just assumed that `call` was valid, but I guess it's a fallout from time when the call wasn't in `emitc` directly.

Anyway, I'll apply suggestion later.

Reply:
It is even more complicated :see_no_evil:  Back in timey we had an `emitc.call` op which was later renamed to `emitc.opaque_call`. After that renaming I introduced a new `emitc.call` op which is more closely resembling `func.func`.

Anyway, thanks for fixing the example in the expression op description which clearly improves the EmitC dialect and lowers the barrier for new users. I appreciate it.